prow/cmd/deck/template/spyglass.html

```diff
@@ -13,11 +13,12 @@
 {{define "content"}}
 {{$source:=.Source}}
 <div id="lens-container">
-  {{if or .JobHistLink .ArtifactsLink .PRHistLink}}
+  {{if or .JobHistLink .ArtifactsLink .PRHistLink .TestgridLink}}
   <div id="links-card" class="mdl-card mdl-shadow--2dp lens-card">
     {{if .JobHistLink}}<a href="{{.JobHistLink}}">Job History</a>{{end}}
     {{if .PRHistLink}}<a href="{{.PRHistLink}}">PR History</a>{{end}}
     {{if .ArtifactsLink}}<a href="{{.ArtifactsLink}}">Artifacts</a>{{end}}
+    {{if .TestgridLink}}<a href="{{.TestgridLink}}">Testgrid</a>{{end}}
```

Comment:
How do you feel about a) making this more generic and b) requiring crier to publish this link?

AKA we write some field into the ProwJob status and/or finished.json metadata that spyglass reads, rather than reading the testgrid config proto

Reply:
In general I'm pretty in favour of having a generic mechanism to attach (multiple) arbitrary links to jobs, which we can then pull out.

At the moment these links have the nice property that we can generate them with no immediate network access, which is a nice property (especially when something has flaked out such that job metadata went missing, which does happen occasionally). That said, while I like it, I'm not super attached to this property.

Is crier likely to be in a position to be able to do this anytime soon?